ReSharper.FSharp/test/data/parsing/arguments/Attribute 05 - No args - parens.gold

```diff
@@ -0,0 +1,6 @@
+﻿open System.ComponentModel
+
+[<Description()>]
```

Comment:
`unit` can actually be a named parameter/argument, at least in F# symbols:
```fsharp
type T() =
    static member M(u: unit) = ()

T.M()
T.M(u = ())
```

Not sure if/how we actually want/need to support it right now, though. :)